src/test/java/at/ac/tuwien/kr/alpha/AnswerSetToXlsxWriterTest.java

```diff
@@ -28,12 +30,13 @@ public void writeAnswerSetFilesTest() throws IOException {
 		writer.accept(0, as);
 		File tmpDirFile = tmpDir.toFile();
 		File[] generatedFiles = tmpDirFile.listFiles();
-		Assert.assertEquals(generatedFiles.length, 1);
+		assertEquals(1, generatedFiles.length);
 		File answerSetFile = generatedFiles[0];
-		Assert.assertEquals("alphaAnswerSet.0.xlsx", answerSetFile.getName());
-		Workbook wb = WorkbookFactory.create(answerSetFile);
-		AnswerSetToWorkbookMapperTest.assertWorkbookMatchesAnswerSet(wb, as);
-		wb.close();
+		assertEquals("alphaAnswerSet.0.xlsx", answerSetFile.getName());
+		try (Workbook wb = WorkbookFactory.create(answerSetFile)) {
+			AnswerSetToWorkbookMapperTest.assertWorkbookMatchesAnswerSet(wb, as);
+			wb.close();
+		}
```

Comment:
Interesting, I didn't know about the try-with-resources statement. My IDE claims that the `.close()` is redundant and can be deleted (in this and all other try-with-resources blocks).
```suggestion
		try (Workbook wb = WorkbookFactory.create(answerSetFile)) {
			AnswerSetToWorkbookMapperTest.assertWorkbookMatchesAnswerSet(wb, as);
		}
```

Reply:
Thanks, I overlooked that - removed obsolete close statements now